package/MDAnalysis/core/topologyattrs.py

```diff
@@ -518,6 +518,16 @@ def set_segments(self, sg, values):
         """Set segmentattributes for a given SegmentGroup"""
         raise NotImplementedError
 
+    @classmethod
+    def are_values_missing(self, values):
```

Comment:
Please use `cls` instead of `self` for classmethod. Also to make it also compatible for `np.nan` see suggestion:

```python
@classmethod
def are_values_missing(cls, values):
    missing_value_label = getattr(cls, 'missing_value_label', 0)

    if missing_value_label is np.nan:
        return np.isnan(values)
    else:
        return values == missing_value_label
```

Reply:
sounds neat!